hack/release.toml

```diff
@@ -103,6 +103,15 @@ These fields were removed from the default machine configuration schema in v1.12
 etcd container image is now pulled from `registry.k8s.io/etcd` instead of `gcr.io/etcd-development/etcd`.
 """
 
+    [notes.grub]
+        title = "GRUB"
+        description = """\
+Talos Linux introduces new machine configuration option `.machine.install.grubUseUKICmdline` to control whether GRUB should use the kernel command line 
```

Comment:
should we actually start a new install multi doc config and add it there, this way we can start slowly moving away from `.machie.install` to a new InstallConfig. This could be even a completely separate document. Feels like we should not add more stuff to `.machine.install`

Reply:
I had discussion with @utkuozdemir about it, and my point is:

* yes, we want to drop `.machine.install`
* if we drop it, we need to replace it with something new (probably it's not even a config, but arguments to the new API call)
* if we want to create new "MachineInstallConfig" document, I would rather move _all_ options into it, rather than introduce a new machine config document with a single flag, while all other flags stay in `.machine.install`

So I feel at this moment, as we're not replacing `.machine.install` for 1.12, the least evil is to add a new field there. If we move to Install API, we don't need this machine config at all.